tailwind.config.js

```diff
@@ -7,8 +7,39 @@ export default {
     "./src/**/*.{js,ts,jsx,tsx}",
   ],
   theme: {
-    extend: {},
-  },
+    colors: {
+      'black': '#1c1c1c',
+      'borders': "#DDDDDD",
+      'bg':'#F3F5F7',
+      'danger-aler-bg': '#FC555619',
+      'unAssigned-bg': "#F7C2D2",
+      'Assigned-bg': "#99BFFB",
+      'InProgress-bg': "#FFE28C",
+      'Completed-bg': "#A1EDB1",
+      'placeholder':'#4D4D4D',
+      'white': '#ffffff',
+      'border-color': '#464646',
+      'purple': '#7e5bef',
+      'pink': '#ff49db',
+      'red':"#CB0132",
+      'orange': '#ff7849',
+      'green': '#13ce66',
+      'yellow': '#ffc82c',
+      'gray-dark': '#273444',
+      'gray': '#8492a6',
+      'gray-light': '#d3dce6',
```

Comment:
rather than gray-dark and gray-light follow numbering convention that tailwind uses.
Ex. instead of gray-light use gray-100 & instead of gray-dark use gray-900
That way use can also other shades of gray in the future.